### PR TITLE
Groupby cleanup

### PR DESCRIPTION
**Pull request by smiklos**
Removing unused parameter.
Count down GROUP_COUNT for each group on complete

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@smiklos Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=reactor/reactor-core&pullRequestId=1093)!

[Click here](https://cla.pivotal.io/sync/reactor?repositoryId=reactor/reactor-core&pullRequestId=1093) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@smiklos Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=reactor/reactor-core&pullRequestId=1093)!

File: reactor-core/src/main/java/reactor/core/publisher/FluxGroupBy.java [hunk 1]
```diff
@@ -224,9 +224,9 @@ public void onComplete() {
 			}
 			for (UnicastGroupedFlux<K, V> g : groupMap.values()) {
 				g.onComplete();
+				GROUP_COUNT.decrementAndGet(this);
```
Comment (akarnokd) [inline reactor-core/src/main/java/reactor/core/publisher/FluxGroupBy.java]: Actually, this decrement is not really necessary as the upstream won't procude further items and thus cancellation has no effect.

File: reactor-core/src/main/java/reactor/core/publisher/FluxGroupBy.java [hunk 2]
```diff
@@ -407,8 +407,7 @@ void drainLoop() {
 
 		boolean checkTerminated(boolean d,
 				boolean empty,
-				Subscriber<?> a,
-				Queue<GroupedFlux<K, V>> q) {
+				Subscriber<?> a) {
```
Comment (akarnokd) [inline reactor-core/src/main/java/reactor/core/publisher/FluxGroupBy.java]: Actually, this parameter should be used instead of the `queue` field.

**Comment by smiklos** [comment]
Another pr then :+1:

**Pull request closed without merging**